lib/stages/hexDump.cpp

```diff
@@ -55,6 +61,11 @@ void hexDump::main_thread() {
         // Prints the hex data to screen
         hex_dump(16, (void*)&frame[_offset], _len);
 
+        // Add sample to trackers
+        static int cnt = 0;
+        tracker_0->add_sample(cnt++);
+        tracker_1->add_sample(cnt << 1);
```

Comment:
It's likely better to make the stat tracker example based on something that's theoretically useful.  This is more of a unit test style example.   Perhaps you could track the time needed in testDataGen to populate a new frame with values, or something like that.